### ISSUE TITLE
Cannot load Zend OPcache - it was already loaded

### ISSUE THREAD

**Issue by dqlopez** [geerlingguy/ansible-role-php#229]
Role installs fine, however, when doing `php -v` it shows a warning along with the PHP version:

```
Cannot load Zend OPcache - it was already loaded
PHP 7.2.2-3+ubuntu14.04.1+deb.sury.org+1 (cli) (built: Feb  6 2018 16:35:00) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.2-3+ubuntu14.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies
```
Looking at the `/etc/php/` it shows it does have two opcache ini files.

**Comment by ak4t0sh** [comment]
Hi try to apply PR #201 ;)

**Comment by geerlingguy** [comment]
PR has been merged, and is in release 3.5.1.
